setup.sh

```diff
@@ -149,3 +171,5 @@ scram b -j ${CORES}
 # extra setup
 cd TreeMaker/Production/test/condorSub/
 python $CMSSW_BASE/src/Condor/Production/python/linkScripts.py
+
+set +x
```

Comment:
should this be inside `if [[ "${DEBUG}" == "true" ]]` ?

Reply:
I can put it there if you'd like, but it doesn't really matter. If the default is `set +x` then doing it again won't make a difference.